microsoft-edge/devtools-guide-chromium/device-mode/index.md

```diff
@@ -287,6 +290,14 @@ To test how quickly your page loads and how it responds at different internet an
 All of the throttling is based upon the normal capability of your laptop or desktop.
 
 
+<!-- ------------------------------ -->
+#### Performance tab has warning icon when throttled
```

Comment:
It's not just the Performance tool that has a warning icon when throttling is enabled in Device Emulation, the Network tool also does.